src/main/java/com/cleanroommc/groovyscript/documentation/Builder.java

```diff
@@ -321,7 +335,9 @@ private String createBuilder(Example example, boolean canBeCommented) {
 
         if (!example.def().isEmpty()) out.append("def ").append(example.def()).append(" = ");
 
-        out.append(reference).append(".").append(builderMethod.getName()).append("()");
+        out.append(reference);
+
+        if (!example.raw()) out.append(".").append(builderMethod.getName()).append("()");
```

Comment:
if this is still needed (im not seeing why it needs to be in the constructor, think thats a relic of the passthrough logic):

would prefer instead having something like this: 
```java
out.append(reference).append(".").append(builderMethod.getName());
out.append("(");
if (!example.params().isEmpty()) out.append(example.params());
out.append(")");
```
where `params` is a `String` with a default of `""`.
as an example, this would do this:
```diff
-@Example(raw = true, value = ".shapelessBuilder('basic').gridInput(item('minecraft:coal'), item('minecraft:iron_ingot')).output(item('minecraft:clay')).maximumTier(1).minimumTier(1)")
+@Example(params = "'basic'", value = ".gridInput(item('minecraft:coal'), item('minecraft:iron_ingot')).output(item('minecraft:clay')).maximumTier(1).minimumTier(1)")
```
another option is to make `value` be an array of strings, and then for each parameter of the method grab the next entry in value. probably overengineering that one.

Reply:
can do the first one, yeah

Reply:
Thinking about it I no longer need the feature. I could just remove the raw, I guess, but I feel like it's fine to keep for housekeeping.